server/client.go

```diff
@@ -1694,6 +1699,7 @@ func (c *client) sendPing() {
 // Assume lock is held.
 func (c *client) generateClientInfoJSON(info Info) []byte {
 	info.CID = c.cid
+	info.ClientIP = c.host
```

Comment:
This is from a previous PR, you should rebase from master first. I will get on a call with you to show you how to do this.